### DISCUSSION TITLE
Welcome to html-to-docx Discussions!

### DISCUSSION THREAD

**Discussion by privateOmega** [privateOmega/html-to-docx#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ammarmohamed2016** [comment]
Can use it for angular 12 project ?
I tried to use it but have lot of error.

**Reply by privateOmega** [reply]
@ammarmohamed2016 Sorry for the delayed response. I havent used angular, could you please create a simple example using html-to-docx, I can try to make html-to-docx work with that and maybe upload to in this repo.
